components/studio/test/shared/studio-enter.ps1

```diff
@@ -36,12 +38,13 @@ try {
       Start-Sleep -Seconds 5
     }
 
-    Send-AwaitCommand "$studio_command rm"
-    
+    Send-AwaitCommand "$studio_command rm -o $studio_name"
+    Receive-AwaitResponse -Stream
+
     # Add the same behavior for `hab studio rm` to ensure that the studio is fully cleaned up before 
     # stopping the Await session.
     $retry = 0
-    while(($retry -lt 10) -and (Test-Path "$studio_name")) {
+    while(($retry -lt 5) -and (Test-Path "$studio_name")) {
```

Comment:
I still like keeping this to a higher number because if we reach the limit, subsequently deleting the studio environment is guaranteed to fail because of open file handles from the hab binary.